src/main/java/org/jitsi_modified/impl/neomedia/rtp/remotebitrateestimator/AimdRateControl.java

```diff
@@ -435,11 +439,38 @@ public void update(RateControlInput input, long nowMs)
                 if (input.incomingBitRate > 0L)
                     timeFirstIncomingEstimate = nowMs;
             }
-            else if (nowMs - timeFirstIncomingEstimate > kInitializationTimeMs
-                    && input.incomingBitRate > 0L)
+            else
             {
-                currentBitrateBps = input.incomingBitRate;
-                bitrateIsInitialized = true;
+                long timeSinceFirstIncomingEstimate = nowMs - timeFirstIncomingEstimate;
+                if (timeSinceFirstIncomingEstimate > kFirstIncomingEstimateExpirationMs)
+                {
+                    if (input.incomingBitRate > 0L)
+                    {
+                        timeFirstIncomingEstimate = nowMs;
+                    }
+                    else
+                    {
+                        timeFirstIncomingEstimate = -1L;
+                    }
+
+                    // FIXME Where do we put this in order to observe it? I see
+                    // two options:
+                    //
+                    // 1. The debug JSON output feels appropriate but that is not
+                    // something that is accessible/plottable in wavefront.
+                    //
+                    // 2. Next best thing is in the videobridge statistics, but
+                    // it feels a bit awkward to put it there. If we want it in
+                    // the jvb stats, how should it be bubbled up? event driven
+                    // or have it pulled?
```

Comment:
@bgrozev @JonathanLennox could you please guys take a look and share your thoughts how to implement tracking for this? Maybe we want to push some of the debug stuff to wavefront?